lib/stoplight/default.rb

```diff
@@ -2,16 +2,9 @@
 
 module Stoplight
   module Default
-    WHITELISTED_ERRORS = [
-      NoMemoryError,
-      ScriptError,
-      SecurityError,
-      SignalException,
-      SystemExit,
-      SystemStackError
-    ].freeze
+    WHITELISTED_ERRORS = [].freeze
 
-    BLACKLISTED_ERRORS = [].freeze
+    BLACKLISTED_ERRORS = [StandardError].freeze
```

Comment:
Although there's only one error in this list, I would prefer to have it on its own like like `WHITELISTED_ERRORS` was.